tests/features/webtoolsAnalytics.feature

```diff
@@ -0,0 +1,32 @@
+@api
+Feature: Check Analytics
```

Comment:
I'm not sure if behat tests are what we need for this module. None of the functionality it provides is based on no UI and requires no input from a site's user so I don't think we could create believable user stories. Instead it seems to me that kernel tests should be more appropriate. cc @pfrenssen

Reply:
Because is about rendering element the Behat seems a valid way to test it.